tests/ArabicdatetimeTest.php

```diff
@@ -115,6 +123,10 @@ public function testGetHijriMonths()
 
     public function testGetArabicHijriMonths()
     {
+        if (version_compare(PHP_VERSION, '5.6', '<')) {
+            $this->markTestSkipped('Trans will not be worked on PHP 5.4!');
```

Comment:
This assertion will not be worked on `php-5.4` and `php-5.5` versions because it seems that the `trans` is not worked.